### ISSUE TITLE
build hangs on graceful shutdown

### ISSUE THREAD

**Issue by InfamousSYN** [InfamousSYN/packer#5]
https://github.com/InfamousSYN/packer/blob/e15bdde240efe520241227d5baa9435ab923f434/templates/kali.json#L38
https://github.com/InfamousSYN/packer/blob/e15bdde240efe520241227d5baa9435ab923f434/templates/kali.json#L72

```
"shutdown_command": "echo {{user `root_password`}} | sudo -S shutdown -P now"
```

**Comment by InfamousSYN** [comment]
locations also affected
https://github.com/InfamousSYN/packer/blob/e15bdde240efe520241227d5baa9435ab923f434/templates/kali-cloud.json#L38
https://github.com/InfamousSYN/packer/blob/e15bdde240efe520241227d5baa9435ab923f434/templates/kali-cloud.json#L72

**Comment by InfamousSYN** [comment]
fixed in commit [f27d60c](https://github.com/InfamousSYN/packer/commit/f27d60c89efde2d99a9aabc35997da857fa455f8)
